modules/update-service-mirror-release.adoc

```diff
@@ -0,0 +1,211 @@
+[id="update-service-mirror-release_{context}"]
+= Mirroring the {product-title} image repository
```

Comment:
The section is not closely connected with section **Installing the OpenShift Update Service Operator**. It would be better to make it an individual section rather than a sub-section of **Installing the OpenShift Update Service Operator**.

Reply:
moved it